From f1b16db48e9dc33a7da9381875010b451ce7eee0 Mon Sep 17 00:00:00 2001
From: Erickson Silva de Oliveira <Erickson.SilvadeOliveira@windriver.com>
Date: Mon, 13 Apr 2026 13:51:40 -0300
Subject: [PATCH] ceph-csi-rbd: replace appVersion/version

This patch changes the appVersion of the rbd chart to the
upstream version that ceph-csi is being updated to.

Signed-off-by: Erickson Silva de Oliveira <Erickson.SilvadeOliveira@windriver.com>
---
 charts/ceph-csi-rbd/Chart.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/charts/ceph-csi-rbd/Chart.yaml b/charts/ceph-csi-rbd/Chart.yaml
index 5ab42a1..041e24e 100644
--- a/charts/ceph-csi-rbd/Chart.yaml
+++ b/charts/ceph-csi-rbd/Chart.yaml
@@ -1,10 +1,10 @@
 ---
 apiVersion: v1
-appVersion: canary
+appVersion: 3.16.2
 description: "Container Storage Interface (CSI) driver,
 provisioner, snapshotter, resizer and attacher for Ceph RBD"
 name: ceph-csi-rbd
-version: 3-canary
+version: 3.16.2
 keywords:
   - ceph
   - rbd
--
2.34.1
